mlir/lib/Catalyst/Transforms/DetectQNodes.cpp

```diff
@@ -514,7 +514,6 @@ void LivenessAnalysisDropRef::rewrite(LLVM::CallOp sink, PatternRewriter &rewrit
     //     llvm.func @mlirAsyncRuntimeAwaitToken(!llvm.ptr)
     //     llvm.func @mlirAsyncRuntimeDropRef(!llvm.ptr, i64)
     auto awaitFnDecl = AsyncUtils::lookupOrCreateAwaitTokenName(moduleOp);
-    auto awaitValFnDecl = AsyncUtils::lookupOrCreateAwaitValueName(moduleOp);
```

Comment:
Are we sure this is safe? @erick-xanadu

Reply:
Yes. The token is always the last one so we don't have to await for the values. I left it there by mistake while trying it out.